src/main/webapp/WEB-INF/tags/student/profile/studentProfileDiv.tag

```diff
@@ -0,0 +1,154 @@
+                   class="form-control"
+                   type="text"
+                   data-actual-value="${profile.nationality}"
+                   value="${profile.nationality}"
+                   placeholder="Nationality">
+        </div>
+        <div class="form-group">
+            <label for="studentGender">
+                Gender
+            </label>
+            <div id="studentGender">
+                <label for="genderMale" class="radio-inline">
+                    <input id="genderMale"
+                           name="<%= Const.ParamsNames.STUDENT_GENDER %>"
+                           class="radio"
+                           type="radio"
+                           value="<%= Const.GenderTypes.MALE %>"
+                           <c:if test="${profile.gender == MALE}">checked="checked"</c:if>> Male
+                </label>
+                <label for="genderFemale" class="radio-inline">
+                    <input id="genderFemale"
+                           name="<%= Const.ParamsNames.STUDENT_GENDER %>"
+                           class="radio"
+                           type="radio"
+                           value="<%= Const.GenderTypes.FEMALE %>"
+                           <c:if test="${profile.gender == FEMALE}">checked="checked"</c:if>> Female
+                </label>
+                <label class="radio-inline" for="genderOther">
+                    <input id="genderOther"
+                           name="<%= Const.ParamsNames.STUDENT_GENDER %>"
+                           class="radio"
+                           type="radio"
+                           value="<%= Const.GenderTypes.OTHER %>"
+                           <c:if test="${profile.gender == OTHER}">checked="checked"</c:if>> Not Specified
+                </label>
+            </div>
+        </div>
+        <div class="form-group"
+             title="<%= Const.Tooltips.STUDENT_PROFILE_MOREINFO %>"
+             data-toggle="tooltip"
+             data-placement="top">
+            <label for="studentNationality">
+                More info about yourself
+            </label>
+            <!-- Do not add whitespace between the opening and closing tags-->
+            <textarea id="studentMoreInfo"
+                      name="<%= Const.ParamsNames.STUDENT_PROFILE_MOREINFO %>"
+                      rows="4"
+                      class="form-control"
+                      placeholder="<%= Const.Tooltips.STUDENT_PROFILE_MOREINFO %>">${profile.moreInfo}</textarea>
+        </div>
+        <br>
+        <button type="submit" id="profileEditSubmit" class="btn btn-primary center-block">
+            Save Profile
+        </button>
+        <br>
+        <p class="text-muted text-color-disclaimer">
+            <i>* This profile will be visible to all your Instructors and Coursemates</i>
+        </p>
+        <input type="hidden" name="<%= Const.ParamsNames.USER_ID %>" value="${profile.googleId}">
```

Comment:
Indeed the field is taken from `data.account.googleId`, but `data` itself is not passed from the main .jsp file to this .tag file.